xl/trax/search.py

```diff
@@ -544,3 +541,25 @@ def match_track_from_string(track, search_string,
     matcher = TracksMatcher(search_string, case_sensitive=case_sensitive,
         keyword_tags=keyword_tags)
     return matcher.match(SearchResultTrack(track))
+
+def normalize_string(string):
```

Comment:
I haven't profiled it, but I have to imagine this is going to make searches really slow since it's running on every file. Can we find a more performant solution?